tests/test_structured_document.py

```diff
@@ -193,52 +193,93 @@ def test_components(self):
         </content>
       </section>
     </body>
+    <attachments>
+      <attachment id="att_1">
+        <doc name="schedule">
+          <meta>
+            <identification source="#slaw">
+              <FRBRWork>
+                <FRBRthis value="/na/act/1977/25/!schedule-A"/>
+                <FRBRuri value="/na/act/1977/25"/>
+                <FRBRalias value="Schedule"/>
+                <FRBRdate date="1980-01-01" name="Generation"/>
+                <FRBRauthor href="#council"/>
+                <FRBRcountry value="na"/>
+              </FRBRWork>
+              <FRBRExpression>
+                <FRBRthis value="/na/act/1977/25/eng@1993-12-02/!schedule-A"/>
+                <FRBRuri value="/na/act/1977/25/eng@1993-12-02"/>
+                <FRBRdate date="1980-01-01" name="Generation"/>
+                <FRBRauthor href="#council"/>
+                <FRBRlanguage language="eng"/>
+              </FRBRExpression>
+              <FRBRManifestation>
+                <FRBRthis value="/na/act/1977/25/eng@1993-12-02/!schedule-A"/>
+                <FRBRuri value="/na/act/1977/25/eng@1993-12-02"/>
+                <FRBRdate date="2020-03-25" name="Generation"/>
+                <FRBRauthor href="#slaw"/>
+              </FRBRManifestation>
+            </identification>
+          </meta>
+          <mainBody>
+            <hcontainer id="schedule" name="schedule">
+              <heading>Schedule</heading>
+              <paragraph id="schedule.paragraph0">
+                <content>
+                  <p>This is the content of the Schedule!</p>
+                </content>
+              </paragraph>
+            </hcontainer>
+          </mainBody>
+        </doc>
+      </attachment>
+    </attachments>
+    <components>
```

Comment:
shouldn't `components` still come after `</act>`?

Reply:
Attachments and components can both come here. They can also both come after act. My motivation here is that this Cobalt object is effectively rooted at the `act` or `doc` or whatever top-level element: therefore `get_components()` should handle anything beneath that, but not outside of it.

In time we can get to working with AKN xml documents that contain multiple document types.